cabal.project

```diff
@@ -2,8 +2,8 @@ packages: .
 
 source-repository-package
     type: git
-    location: https://github.com/luc-tielen/llvm-codegen.git
-    tag: a783cb933146d8f95282e7ec207e5510bc36e3ce
+    location: https://github.com/placidex/llvm-codegen.git
```

Comment:
I merged that PR, can you point back to latest commit in my repo again? Thanks!

Reply:
done, thanks.